utils/github_utils.py

```diff
@@ -0,0 +1,11 @@
+import requests
+
+from github import Github
+from flask import current_app as app
+from constants import MASADER_GH_REPO
+
+def report_issue(title: str, msg:str):
```

Comment:
try to use full names instead of shortcuts unless it is required (e.g. `message: str`, instead of `msg:str`).